XamarinCommunityToolkit/Views/CameraView/CameraView.shared.cs

```diff
@@ -14,27 +14,17 @@ public class CameraView : View
 
 		internal event EventHandler ShutterClicked;
 
-		public static readonly BindableProperty ShutterCommandProperty =
-			BindableProperty.Create(nameof(ShutterCommand),
-						typeof(ICommand),
-						typeof(CameraView));
+		internal static readonly BindablePropertyKey ShutterCommandPropertyKey =
+			BindableProperty.CreateReadOnly(nameof(ShutterCommand),
+				typeof(ICommand),
+				typeof(CameraView),
+				default,
+				BindingMode.OneWayToSource,
```

Comment:
This is the correct `BindingMode`! This part is very important to get right.

Good Job 🎉